builders/relayedTxV1Builder.go

```diff
@@ -112,6 +112,7 @@ func prepareInnerTxForRelayV1(tx *data.Transaction) (string, error) {
 		return "", err
 	}
 
+	// TODO: remove this hardcoded implementation in core. Create there a dedicated converter between FrontendTransaction <-> Transaction
```

Comment:
// TODO: remove this hardcoded implementation. Inside mx-chain-core-go, create there a dedicated converter between FrontendTransaction <-> Transaction

Reply:
done